model/Models/Column.cs

```diff
@@ -55,11 +55,22 @@ public Column(string name, string type, byte precision, int scale, bool nullable
 			}
 		}
 
-		public string RowGuidColText {
-			get { return IsRowGuidCol ? "ROWGUIDCOL" : string.Empty; }
-		}
-
-		public ColumnDiff Compare(Column c) {
+        public string RowGuidColText {
+	        get { return IsRowGuidCol ? "ROWGUIDCOL" : string.Empty; }
+        }
+
+        public string Collation { get; set; }
+
+        public string CollationText {
```

Comment:
Nitpick but `Collation` and `CollationText` could just be one property with a backing field.